buildSrc/src/main/java/com/synopsys/integration/detect/docs/content/Terms.java

```diff
@@ -7,13 +7,13 @@ public class Terms {
     private final Map<String, String> termMap = new HashMap<>();
 
     public Terms() {
-        termMap.put("solution_name", "Synopsys Detect");
+        termMap.put("solution_name", "Black Duck Detect");
         termMap.put("script_repo_url_bash", "https://detect.synopsys.com/detect9.sh");
-        termMap.put("script_repo_url_powershell", "https://detect.synopsys.com/detect9.ps1");
-        termMap.put("binary_repo_url_project", "https://repo.blackduck.com/bds-integrations-release/com/blackduck/integration/synopsys-detect");
-        termMap.put("binary_repo_ui_url_project", "https://repo.synopsys.com/bds-integrations-release/com/synopsys/integration/synopsys-detect");
-		termMap.put("binary_repo_jenkins_url_project", "https://repo.blackduck.com/bds-integrations-release/com/blackducksoftware/integration/blackduck-detect/");
-		termMap.put("binary_repo_url_sigma", "https://repo.blackduck.com/sigma-release-trial/2022.6.0/");
+        termMap.put("script_repo_url_powershell", "https://detect.blackduck.com/detect9.ps1");
+        termMap.put("binary_repo_url_project", "https://repo.blackduck.com/bds-integrations-release/com/blackduck/integration/detect");
+        termMap.put("binary_repo_ui_url_project", "https://repo.blackduck.com/bds-integrations-release/com/blackduck/integration/detect");
+	termMap.put("binary_repo_jenkins_url_project", "https://repo.blackduck.com/bds-integrations-release/com/blackducksoftware/integration/blackduck-detect/");
+	termMap.put("binary_repo_url_sigma", "https://repo.blackduck.com/sigma-release-trial/2022.6.0/");
```

Comment:
Not related to rebranding, but is that still valid? 
https://repo.blackduck.com/sigma-release-trial/2022.6.0 ?

Reply:
Yes this is still valid, used for sig iac scans

Reply:
Shall I update it to latest? https://repo.blackduck.com/sigma-release-trial/2024.9.0/

Reply:
@gopannabd Please no! This is not in scope for 10.0.0 to ensure new version is completely compatible yet

Reply:
Booo - safety first I suppose.

Reply:
@sig-cpotts made me do it!